src/unix/luv/dune

```diff
@@ -0,0 +1,18 @@
+(* -*- tuareg -*- *)
+
+let preprocess =
+   match Sys.getenv "BISECT_ENABLE" with
+   | "yes" -> "(preprocess (pps bisect_ppx))"
+   | _ -> ""
+   | exception _ -> ""
+
+let () = Jbuild_plugin.V1.send @@ {|
+
+(library
+ (name lwt_luv)
```

Comment:
IIRC `(name ...)` is redundant if `(public_name ...)` is defined nowadays. We only have both fields in other `dune` files because it's older code.